ansible_collections/arista/avd/roles/eos_designs/templates/network_services/ip-igmp-snooping.j2

```diff
@@ -13,10 +13,13 @@ ip_igmp_snooping:
 {# Detect if a svi_profile exists #}
 {# If exists, create a shortpath to access profile data #}
 {%                 if svi.profile is arista.avd.defined %}
-{%                     set svi_profile = svi_profiles[svi.profile] | arista.avd.default() %}
+{%                     set svi_profile = svi_profiles | arista.avd.default({}) |
```

Comment:
Why is `arista.avd.default` used with `{}` as a default value when `svi_profiles` is actually a `list` of dictionaries?

Reply:
its before convert_dicts

Reply:
ah yes makes sense - thanks!